### ISSUE TITLE
[Bug] Able to take Screenshots  on iOS (not on Android)

### ISSUE THREAD

**Issue by NetScr1be** [XRPL-Labs/XUMM-Issue-Tracker#282]
**Describe the bug**

Client reports Wife and father are both on iPhone and are able to take screenshots. He is on Android and screenshots are disabled

N.B. Client in [another issue](https://github.com/XRPL-Labs/XUMM-Issue-Tracker/issues/281) also able to provide screenshot

**To Reproduce**

1. Open XUMM
2. take screenshot

**Expected behavior**

Screenshots disabled

**Screenshots**
If applicable, add screenshots or a video to help explain your problem.

### Environment

* XUMM version: [Latest (downloaded in the week prior to report)]

What device are you running XUMM on?

* Manufacturer: [Apple]
* Model: [iphone 7]
* OS version: [4.0.1]
* Platforms
  * [X] iOS
  * [ ] Android

### Additional context

 > Hello. I noticed on my Andriod device that I'm not allowed to take screenshots when XUMM is open. 

My wife has an iPhone. She is able to take screenshots no problem. Is that intentional or a bug?

**Link:** [Conversation with delayedgratificationdisciple@gmail.com](https://app.intercom.com/a/apps/nq9rb5du/respond/inbox/all/conversations/152601500000637)

**Customer:** [delayedgratificationdisciple@gmail.com](https://app.intercom.com/a/apps/nq9rb5du/users/5f9dc6c95e42380f99162b21/all-conversations)
**Customer ID:** 93151300-3c9a-4ca7-a578-bd642c2d3480
___
Intercom Operator will send a note to [this conversation](https://app.intercom.com/a/apps/nq9rb5du/respond/inbox/all/conversations/152601500000637) when this issue is closed.

**Comment by RetirePerspired** [comment]
Looks like my father is running an Iphone XR with version 13.5.1

Also slight correction in the ticket. The above iphone version should be 14.0.1, not 4.0.1 :)

Thanks for taking a look at this!

**Comment by WietseWind** [comment]
Correct, screenshots are only blocked on Android, not on iOS. iOS doesn't support blocking this. On Android, we'll add an option in the upcoming beta that allows users to re enable screenshots (eg. for making tutorials, etc.)
